.clabot

```diff
@@ -35,7 +35,9 @@
     "wonderwomancode",
     "devabdee",
     "Ether1oop",
-    "omahs"
+    "omahs",
+    "0xYee",
+    "SolomonFoskaay"
```

Comment:
Hum? Please don't change the `.clabot` config :D

Reply:
mhh, let me check, i think there is something wrong with the rebase